src/xdocs/config_misc.xml

```diff
@@ -1157,12 +1174,44 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
 
       <subsection name="Examples" id="Indentation_Examples">
         <p>
-          To configure the check:
+          To configure the check for default behavior:
         </p>
         <source>
 &lt;module name=&quot;Indentation&quot;/&gt;
         </source>
-
+        <p>
+          Example of Compliant code for default configuration
+          (in comment name of property that controls indentations):
+        </p>
+        <source>
+class Test {
+    String field;               // basicOffset
+    int[] arr = {               // basicOffset
+        5,                      // arrayInitIndent
+        6 };                    // arrayInitIndent
+    void bar() throws Exception // basicOffset
+    {                           // braceAdjustment
+        foo();                  // basicOffset
+    }                           // braceAdjustment
+    void foo() {                // basicOffset
+        if ((cond1 &amp;&amp; cond2)  // basicOffset
+                  || (cond3 &amp;&amp; cond4)    // lineWrappingIndentation and forceStrictCondition
+                  ||!(cond5 &amp;&amp; cond6)) { // lineWrappingIndentation and forceStrictCondition
+            field.doSomething()        // basicOffset
+                .doSomething()         // lineWrappingIndentation and forceStrictCondition
+                .doSomething( c -> {   // lineWrappingIndentation and forceStrictCondition
+                    return c.doSome(); // basicOffset
+                });
+        }
+    }
+    void fooCase()                // basicOffset
+        throws Exception {        // throwsIndent
+        switch (field) {          // basicOffset
+            case "value" : bar(); // caseIndent
+        }
```

Comment:
I assume this line is still `basicOffset`?

Reply:
braceAdjustment or baseOffcet .... I tried to skip information that does not matter here. It is example for `case`